tubular/nominal.py

```diff
@@ -266,6 +276,7 @@ def __init__(
         weight=None,
         rare_level_name="rare",
         record_rare_levels=True,
+        encode_unseen_levels: bool = True,
```

Comment:
Nit -  `unseen_levels_to_rare` might be more explicit than `encode_unseen_levels`

Reply:
Yeah I agree that reads a lot more clearly. Done here:

https://github.com/lvgig/tubular/pull/141/commits/d1dd04bdff87b19a48bd449aa04eba6ff44e0a23